### PR TITLE
Fix broken contextual panels in accordion

### PR DESCRIPTION
**Pull request by nireno**
## Description
This fixes issue [#511](https://github.com/patternfly/patternfly/issues/511) which caused contextual panels to break when used inside .panel-group. 

@andresgalante

### CODE REVIEW

**Comment by andresgalante** [comment]
Thanks a lot @nireno, this is great! 😄 
I wonder if we need gradients for on those headers to keep consistency. Let me loop @kybaker on this one:
Kyle, can you take a look at the contextual panels: https://rawgit.com/andresgalante/patternfly/panel-color-dist/dist/tests/accordions.html 
 
We are adding the option to select different colors on panels. It's something that bootstrap has but we were overwriting it: http://getbootstrap.com/components/#panels-alternatives

@nireno can we add the examples to panels instead of accordion please?

**Comment by kybaker** [comment]
@nireno @andresgalante - The gradient should be avoided for panels. You should also use the border and background colors from the notifications: http://www.patternfly.org/pattern-library/communication/inline-notifications/#/api

**Comment by andresgalante** [comment]
@nireno to change the colors please add new variables on our variables.less to overwrite the default ones:

```
@panel-success-text:          @state-success-text;
@panel-success-border:        @state-success-border;
@panel-success-heading-bg:    @state-success-bg;

@panel-info-text:             @state-info-text;
@panel-info-border:           @state-info-border;
@panel-info-heading-bg:       @state-info-bg;

@panel-warning-text:          @state-warning-text;
@panel-warning-border:        @state-warning-border;
@panel-warning-heading-bg:    @state-warning-bg;

@panel-danger-text:           @state-danger-text;
@panel-danger-border:         @state-danger-border;
@panel-danger-heading-bg:     @state-danger-bg;
```
Thanks a lot!! 😄

File: src/less/variables.less [hunk 1]
```diff
@@ -394,17 +394,30 @@
 @nav-pf-vertical-badges-width:                                      (@nav-pf-vertical-width + 50px);
 @padding-small-horizontal:                                          @padding-base-horizontal;
 @padding-small-vertical:                                            @padding-base-vertical;
-@panel-danger-border:                                               @brand-danger;
+@panel-danger-bg:                                                   @state-danger-bg;
```
Comment (andresgalante) [inline src/less/variables.less]: Hi @nireno, you don't really need this one here. By changing just`@state-danger-bg` then `@panel-danger-bg` will change.

Same on the other` panel-` variables

**Comment by andresgalante** [comment]
@nireno so two things: First Thank you very much, I've just reviewed everything and it looks ok to me.

Second: I am really sorry, but you are right this belongs to accordion and not to panels. Sorry :( Can we move it back? I can send you a PR if you don't have time.

Thanks a lot, and again I am sorry for it. I am ready to merge it once we move it.

**Comment by nireno** [comment]
Don't worry I saw this coming :). I'll take care of it.

**Comment by andresgalante** [comment]
Lets merge this one. thanks a lot @nireno !

**Comment by bleathem** [comment]
Thanks for the contribution @nireno !

**Comment by nireno** [comment]
My first pull request accepted? I can die in peace :)

**Comment by andresgalante** [comment]
@nireno sorry I didn't make it easy for you, thanks a lot for your patience and contribution 😄

**Comment by LHinson** [comment]
thank you @nireno! 

bitmoji

**Pull request merged**